File: Aide.ClientApp/src/app/models/insurance-probatory-document.ts
import { ProbatoryDocument } from './probatory-document';
import { ClaimTypeId } from '../enums/claim-type-id.enum';

export class InsuranceProbatoryDocument {
    id: number;
    insuranceCompanyId: number;
    claimTypeId: ClaimTypeId;
    probatoryDocumentId: number;
    sortPriority: number;
    groupId: number;
    dateCreated?: string;
    dateModified?: string;
    probatoryDocument?: ProbatoryDocument;
}

File: Aide.Claims.Domain/Objects/InsuranceProbatoryDocument.cs
using Aide.Claims.Domain.Enumerations;
using System;

namespace Aide.Claims.Domain.Objects
{
    public class InsuranceProbatoryDocument
    {
        public int Id { get; set; }
        public int InsuranceCompanyId { get; set; }
        public EnumClaimTypeId ClaimTypeId { get; set; }
        public int ProbatoryDocumentId { get; set; }
        public int SortPriority { get; set; }
        public int GroupId { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateModified { get; set; }

        public ProbatoryDocument ProbatoryDocument { get; set; }
    }
}
